_pages/index.html

```diff
@@ -10,13 +10,14 @@
     <!-- Container 1  (5 protein morning snacks) -->
     <div> 
         <h1>
-            <strong>
-            15 Protein-Packed Meals That Are 
+           <p class="Title">
```

Comment:
Do you need a paragraph element inside a heading element? Why not give h1 the title class?